cli/flags.rs

```diff
@@ -1014,6 +1026,19 @@ Ignore linting a file by adding an ignore comment at the top of the file:
         .long("rules")
         .help("List available rules"),
     )
+    .arg(
+      Arg::with_name("json")
+        .long("json")
+        .help("Output lint result in JSON format.")
+        .takes_value(false),
+    )
+    .arg(
+      Arg::with_name("sort")
+        .long("sort")
+        .help("Sort the lints of the outputted JSON.")
+        .requires("json")
```

Comment:
**note:** Limiting the use of `--sort` to `--json` because we don't want or need to sort with the console pretty print reporter.